developer/introduction.rst

```diff
@@ -0,0 +1,258 @@
+Introduction to Python and to the Django world
+==============================================
+
+This page will introduce you to python, to some of python's modules such
+as ipython and to the django and django-rest frameworks.
+
+Python and its advantages
+-------------------------
+
+As you may already know, python is a programming language, more
+specifically an interpreted high-level programming language for
+general-purpose programming. It is also widely used these days!
+Big companies like Google, Disney and Dropbox make excessive use
+of this programming language! You'll know why in an instant.
+
+- To begin with, the syntax in Python helps programmers to do
+  coding in fewer steps as compared to other languages such as
+  Java or C++ because it uses relatively less lines and words
+  while still performing the same operations.
+- Additionally, python offers excellent readability and uncluttered
+  simple-to-learn syntax which helps beginners to utilize this
+  programming language.
+- Also, python has a large standard library full of pre-defined
+  functions which can help fulfil tasks related to string operations,
+  Internet, web service tools, operating system interfaces and protocols
+  a lot more quickly and easily.
+- Believe it or not, python language is developed under an OSI-approved
+  open source license, which makes it free to use and distribute,
+  including for commercial purposes!
+- Python has built-in list and dictionary data structures which can be
+  used to construct fast runtime data structures too!
+- Finally, python has clean object-oriented design, provides enhanced
+  process control capabilities, and possesses strong integration and
+  text processing capabilities and its own unit testing framework,
+  all of which contribute to the increase in its speed and productivity.
```

Comment:
please add something about the fact that finding developers who know python is very easy and that allows the project to expand, also mention python is very widely used in the networking and configuration management world (cite famous libraries like networkx, ansible, salt, paramiko, fabric